runtime/gc_base/gc_internal.h

```diff
@@ -89,7 +89,7 @@ extern J9_CFUNC void j9gc_objaccess_indexableStoreI16(J9VMThread *vmThread, J9In
 extern J9_CFUNC void j9gc_all_object_and_vm_slots_do(J9JavaVM *javaVM, void *function, void *userData, UDATA walkFlags);
 extern J9_CFUNC j9object_t j9gc_objaccess_referenceGet(J9VMThread *vmThread, j9object_t refObject);
 extern J9_CFUNC UDATA j9gc_ext_check_is_valid_heap_object(J9JavaVM *javaVM, j9object_t ptr, UDATA flags);
-extern J9_CFUNC void J9WriteBarrierJ9ClassBatchStore(J9VMThread *vmThread, J9Class *destinationJ9Class);
+extern J9_CFUNC void J9WriteBarrierClassBatch(J9VMThread *vmThread, J9Class *destinationJ9Class);
```

Comment:
It probably makes sense to simplify the parameter name (destinationJ9Class -> destinationClass, or destinationClazz what is commonly used to avoid confusion with C++ keyword 'class') as well.
I would not worry about actual argument names at  call sites (since it may involve changing more coded outside of GC scope), but just parameters in function declaration/definition points.
Up to you if you really want to change it, but If you do change , check other similar functions, too.

Reply:
Param names simplified, summary of changes:

**`J9WriteBarrierClassBatch`**
```diff
accessBarrier.cpp
- J9WriteBarrierClassBatch(J9VMThread *vmThread, J9Class *dstJ9Class)
+ J9WriteBarrierClassBatch(J9VMThread *vmThread, J9Class *dstClazz)

gc_internal.h
- extern J9_CFUNC void J9WriteBarrierClassBatch(J9VMThread *vmThread, J9Class *destinationJ9Class);
+ extern J9_CFUNC void J9WriteBarrierClassBatch(J9VMThread *vmThread, J9Class *destinationClazz);

j9nonbuilder.h
- void  ( *J9WriteBarrierClassBatch)(struct J9VMThread *vmThread, J9Class *destinationJ9Class) ;
+ void  ( *J9WriteBarrierClassBatch)(struct J9VMThread *vmThread, J9Class *destinationClazz) ;
```

**`J9WriteBarrierPostClass`**
```diff
gc_internal.h
- extern J9_CFUNC void J9WriteBarrierPostClass(J9VMThread *vmThread, J9Class *destinationJ9Class, j9object_t storedObject);
+ extern J9_CFUNC void J9WriteBarrierPostClass(J9VMThread *vmThread, J9Class *destinationClazz, j9object_t storedObject);

j9nonbuilder.h
- void  ( *J9WriteBarrierPostClass)(struct J9VMThread *vmThread, J9Class *destinationJ9Class, j9object_t storedObject) ;
+ void  ( *J9WriteBarrierPostClass)(struct J9VMThread *vmThread, J9Class *destinationClazz, j9object_t storedObject) ;
```

Other similar function look fine